test/black/hct-hott.t/run.t

```diff
@@ -411,9 +411,48 @@
    ￫ info[I0000]
    ￮ constant 𝕗_𝕎_code defined
   
+   ￫ info[I0000]
+   ￮ constant 𝕗𝕎 definedd
```

Comment:
```suggestion
   ￮ constant 𝕗𝕎 defined
```

Reply:
Did you use `dune promote` to update the `run.t`?  If so, it should make the output correct automatically.